src/CmsKitDemo/CmsKitDemoAssemblyInfo.cs

```diff
@@ -74,7 +74,7 @@ public static AssemblyInformation GetAssemblyInformationOrNull()
     {
         try
         {
-            var assembly = Assembly.GetAssembly(typeof(AbpModule));
+            var assembly = Assembly.GetAssembly(typeof(CmsKitDemoModule));
```

Comment:
Will the `AssemblyInformationalVersionAttribute` in the `CmsKitDemoModule` assembly provide the correct ABP version?

Reply:
![image](https://github.com/user-attachments/assets/8b6ed9bc-714b-46f4-a280-86dc34fc4569)
abp version is wrong

Reply:
https://github.com/abpframework/cms-kit-demo/pull/20/commits/7ab9a1f6363eff6971d9e545ef92f95a7235158d